core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties

```diff
@@ -0,0 +1,33 @@
+title=Manage Legacy API Token usage
+contextMessage=The following users are using Legacy API Token. \
+As that old system stores the credentials in a recoverable format, it is strongly advised to migrate to the new system.<br /> \
+In previous versions of Jenkins, the creation of a new user implied to the generation of an Legacy API Token. \
+Often majority of users are not using that token and so provide a larger potential attack surface than required.<br /> \
+From now, the automatic generation of token is determined by configuration.
+recommendationMessage=For such reasons, we recommend to: \
+<ul>\
+  <li>revoke the tokens that were never used and</li>\
+  <li>ask users that are effectively using their legacy token to generate a new token using the new system and revoking the legacy one</li>\
```

Comment:
nit: I would reword this to "ask users that are currently using their legacy token to generate a token using the new system and revoke their legacy token"